### PR TITLE
chore: remove private dependencies from aggregate packages

### PR DESCRIPTION
**Pull request by sam-goodwin**
@functionless/test (which is a private package) was in our public package's dependencies

- [x] remove `@functionles/test`
- [x] stop running doctor scripts on `yarn install`

### CODE REVIEW

File: apps/test-app/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@graphql-codegen/typescript-resolvers": "^2.6.1",
     "@graphql-codegen/typescript": "^2.4.8",
     "@types/node": "^16",
-    "functionless": "^0.27.1",
```
Comment (thantos) [inline apps/test-app/package.json:19]: need to merge tyler's change?

File: apps/website/package.json [hunk 2]
```diff
@@ -104,7 +103,11 @@
     "@types/react": "^17.0.0"
   },
   "browserslist": {
-    "production": [">0.5%", "not dead", "not op_mini all"],
+    "production": [
+      ">0.5%",
+      "not dead",
+      "not op_mini all"
+    ],
```
Comment (thantos) [inline apps/website/package.json:110]: this is becoming a problem, prettier and lerna or some script keep re-formatting json documents.

File: apps/test-app/package.json (hunk 1 above)
Comment (tvanhens) [inline apps/test-app/package.json:19]: is this intentional?

Comment (sam-goodwin) [reply apps/test-app/package.json:19]: it is not

File: scripts/doctor-aggregate-packages.mjs [hunk 3]
```diff
@@ -68,6 +68,7 @@ async function patch(pkgJsonPath, pkgJson, packages, dependenciesKey) {
     ...(pkgJson[dependenciesKey] ?? {}),
     ...Object.fromEntries(
       packages
+        .filter((pkg) => pkg.private !== true)
```
Comment (thantos) [inline scripts/doctor-aggregate-packages.mjs:71]: Should this be based on if the current project is itself private?

File: apps/test-app/package.json (hunk 1 above)
Comment (sam-goodwin) [reply apps/test-app/package.json:19]: weird, that's not in my local

**Review by thantos — CHANGES_REQUESTED** [review]

File: apps/test-app/package.json (hunk 1 above)
Comment (sam-goodwin) [reply apps/test-app/package.json:19]: Oh actually, it was intentional - it was causing a problem where differnet versions of functionless were installed from NPM and not fetched from the monorepo

Comment (sam-goodwin) [reply apps/test-app/package.json:19]: Fixed.

File: apps/website/package.json (hunk 2 above)
Comment (sam-goodwin) [reply apps/website/package.json:110]: I disabled them. They will only run manually now.

File: scripts/doctor-aggregate-packages.mjs (hunk 3 above)
Comment (sam-goodwin) [reply scripts/doctor-aggregate-packages.mjs:71]: That could make sense but the script is really just to automate the docs and aggregate functionless package. We can remove this doctor script probably.

File: apps/website/package.json (hunk 2 above)
Comment (sam-goodwin) [reply apps/website/package.json:110]: This new formatting is what VS code does automatically. We should check it in. It shouldn't auto change any more.

**Review by thantos — APPROVED** [review]

**Pull request merged**